packages/components/src/components/Address/Address.test.tsx

```diff
@@ -6,7 +6,7 @@ import { Address } from './Address';
 
 /**
  * We need to mock the Clipboard API by creating a global navigator object.
- * 
+ *
```

Comment:
Can you revert all the changes in this file? I think it's not related to your PR. Let's keep the Git history and blame clean!

Reply:
@hone1er no you're good haha